Misc/NEWS.d/next/Library/2019-05-09-18-50-55.bpo-35070.4vaqNL.rst

```diff
@@ -0,0 +1 @@
+posix.getgrouplist() fails when when the user has NGROUPS_MAX supplemental GIDs. This becomes a problem on OS X Mojave (14.x) because the system defines NGROUPS_MAX to be 16 and it is common for a user to belong to 16 groups. The fix is to increase the size of the array passed to the underlying getgrouplist() POSIX API function to allow space for the user's primary group, which is not accounted for in the NGROUPS_MAX constant (per POSIX.1).
```

Comment:
Thanks for adding the blurb entry.  It's a bit too verbose, though.  It should only be one or two sentences and just needs to say something about preventing test_getgrouplist failures on macOS.  If anyone needs to find more detaill, they can refer to the code.  Also, this problem is not new to macOS 10.14; it only depends on how many groups the user is a member of.  So the test may pass on 10.14 and may fail on older systems.